test/test_type_hints.py

```diff
@@ -157,7 +157,8 @@ def is_torch_mypyini(path_to_file):
                     self.skipTest("Can't find PyTorch MyPy config file")
 
                 import numpy
-                if numpy.__version__.startswith('1.20.0.dev0'):
+                # Mypy thinks that NumPy has no attribute "__version__" even though it has
```

Comment:
This was an omission in numpy 1.20.0, fixed in numpy 1.20.1 - can you add that to the comment so it can easily be cleaned up in the future?